platform-coop-toolkit/coop-map-directory-index
# feat: add latitude and longitude fields to user model

## Description

The `User` model does not have `lat` and `lng` float fields like the Organization does. This PR adds them so that when editing Users, the saved values can be retrieved easily.

## Steps to test

1. Check out branch.
2. Run migrations.
3. Create an individual profile, then visit it in the admin UI.

**Expected behavior:** See that the `User.lat` and `User.lng` fields appear in the Admin UI, and that creating and saving an individual profile saves latitude/longitude values to these fields as well as populating the `User.geom` field.

## Additional information

Not applicable.

## Related issues

Not applicable.
